jekyll/_cci2/customizations.adoc

```diff
@@ -134,3 +134,80 @@ export CIRCLE__OUTER__LOGIN_BANNER_MESSAGE="<insert-your-message-here>
 
 .Login Screen Banner Example
 image::banner.png[]
+
+=== Resource Classes
+You can customize resource classes for your installation to provide developers with https://circleci.com/docs/2.0/optimizations/#resource-class[CPU/RAM options] for the Jobs they configure.
+
+TIP: Once resource classes are set using the steps below, make these options available to developers so they can ensure correct usage.
+
+Following are the steps required to customize resource classes for the Docker and `machine` executors:
+
+. SSH into the Services machine.
+. Run the following:
++
+```shell
+sudo mkdir /etc/circleconfig/picard-dispatcher
+```
+. Run the following:
++
+```shell
+sudo vim /etc/circleconfig/picard-dispatcher/resource-definitions.edn
+```
+. Add your required customizations to the file, then save and exit vim with `:wq` - see below for options and formatting.
++
+WARNING: It is important to use the IDs listen in the example below for the `machine` resources classes. IDs can, however, be changed for Docker resource classes.
```

Comment:
I'm wondering if we should actually phrase this to just tell people not to change it. I can see how the machine parts would be confusing but realistically I'd rather not put the burden on support against seeing what weird thing people would change in the machine stanzas.

Reply:
Yeah agreed

Reply:
done